### PR TITLE
Logging from the generators

### PR DESCRIPTION
**Pull request by johanandren**
Refs #257

Had to do some unethical trix with sbt, but in maven it was no funky business. Didn't check how to deal with it in gradle yet.

### CODE REVIEW

**Comment by johanandren** [comment]
Failure is the not-looked-into-yet gradle

**Comment by johanandren** [comment]
Couldn't get logging from the generators to work in gradle, since they execute in a different way, so instead those are just silenced for now. Tried making those write to a file and then reading that afterwards but couldn't get it to work for some reason (lacking understanding of gradle most likely)

**Comment by johanandren** [comment]
This one is ready for review. 

When run from the gradle plugin the logs are now written to a file which is then read and logged by a task after the protoc plugin is run.

File: codegen/src/main/scala/akka/grpc/gen/Logging.scala [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  }
+  def warn(text: String): Unit = {
+    printer.println(s"[warn] $text")
+    printer.flush()
+  }
+  def error(text: String): Unit = {
+    printer.println(s"[error] $text")
+    printer.flush()
+  }
+}
```
Comment (chbatey) [inline codegen/src/main/scala/akka/grpc/gen/Logging.scala]: Dreaded new line

File: codegen/src/main/scala/akka/grpc/gen/scaladsl/play/PlayScalaClientCodeGenerator.scala [hunk 2]
```diff
@@ -18,25 +19,23 @@ object PlayScalaClientCodeGenerator extends ScalaCodeGenerator {
...
     if (allServices.nonEmpty) {
       val packageName = packageForSharedModuleFile(allServices)
       val b = CodeGeneratorResponse.File.newBuilder()
       b.setContent(AkkaGrpcClientModule(packageName, allServices).body)
       b.setName(s"${packageName.replace('.', '/')}/${ClientModuleName}.scala")
       val set = Set(b.build)
-      /* FIXME we cannot stdout from the generator, it breaks things #257
-      println(s"Generated [${packageName}.${ClientModuleName}] add it to play.modules.enabled and a section " +
-        "with Akka gRPC client config under akka.grpc.client.[servicepackage.ServiceName] to be able to inject " +
+      logger.info(s"Generated [${packageName}.${ClientModuleName}] add it to play.modules.enabled and a section " +
```
Comment (chbatey) [inline codegen/src/main/scala/akka/grpc/gen/scaladsl/play/PlayScalaClientCodeGenerator.scala]: :+1:

File: plugin-tester-java/build.gradle [hunk 3]
```diff
@@ -10,7 +10,7 @@ buildscript {
     //classpath 'gradle.plugin.com.lightbend.akka.grpc:akka-grpc-gradle-plugin:0.1'
 
     // For testing on travis, we use another group id:
-    classpath 'com.lightbend.akka.grpc:akka-grpc-gradle-plugin:SNAPSHOT'
+    classpath 'com.lightbend.akka.grpc:akka-grpc-gradle-plugin:0.1+80-14276983'
```
Comment (chbatey) [inline plugin-tester-java/build.gradle]: this intentional?

File: plugin-tester-scala/build.gradle [hunk 4]
```diff
@@ -10,7 +10,7 @@ buildscript {
     //classpath 'gradle.plugin.com.lightbend.akka.grpc:akka-grpc-gradle-plugin:0.1'
 
     // For testing on travis, we use another group id:
-    classpath 'com.lightbend.akka.grpc:akka-grpc-gradle-plugin:SNAPSHOT'
+    classpath 'com.lightbend.akka.grpc:akka-grpc-gradle-plugin:0.1+80-39d48e3b'
```
Comment (chbatey) [inline plugin-tester-scala/build.gradle]: and this?

File: plugin-tester-java/build.gradle (hunk 3 above)
Comment (johanandren) [reply plugin-tester-java/build.gradle]: Ups. no, certainly not, thanks!

**Comment by johanandren** [comment]
Merged to move forward while we are so short on people.

**Pull request merged**